localstack/testing/pytest/fixtures.py

```diff
@@ -1800,33 +1794,38 @@ def cleanups():
 
 
 @pytest.fixture(scope="session")
-def region(aws_client):
+def account_id(aws_client):
     if is_aws_cloud() or is_api_enabled("sts"):
-        return aws_client.sts.meta.region_name
+        return aws_client.sts.get_caller_identity()["Account"]
     else:
-        return TEST_AWS_REGION_NAME
+        return constants.TEST_AWS_ACCOUNT_ID
 
 
 @pytest.fixture(scope="session")
-def account_id(aws_client):
+def region_name(aws_client):
     if is_aws_cloud() or is_api_enabled("sts"):
-        return aws_client.sts.get_caller_identity()["Account"]
+        return aws_client.sts.meta.region_name
     else:
-        return TEST_AWS_ACCOUNT_ID
+        return constants.TEST_AWS_REGION_NAME
 
 
 @pytest.fixture(scope="session")
 def secondary_account_id(secondary_aws_client):
     if is_aws_cloud() or is_api_enabled("sts"):
         return secondary_aws_client.sts.get_caller_identity()["Account"]
     else:
-        return SECONDARY_TEST_AWS_ACCOUNT_ID
+        return constants.SECONDARY_TEST_AWS_ACCOUNT_ID
+
+
+@pytest.fixture(scope="session")
+def secondary_region_name():
+    return constants.SECONDARY_TEST_AWS_REGION_NAME
```

Comment:
Not a particular fan of using the constants for the fixtures. One of the longer-running efforts will be moving to a state where we won't have access to localstack internals from the test code (e.g. because we're running LS in a container and test against that). I also think `constants` should generally not be a place where we put values used in testing.

How about we use environment variables to configure these and falling back to a static default in the fixture?

Reply:
Actually, this value already gives precedence to the env variable

https://github.com/localstack/localstack/blob/unify-fixtures/localstack/constants.py#L159-L163

I admit that its place in `localstack.constants` is misleading.

Would it be okay if I add a todo note and make the changes in a follow up PR, given how large this PR is already?